oci/oci.go

```diff
@@ -275,7 +275,7 @@ func (r *Runtime) ExecSync(c *Container, command []string, timeout int64) (resp
 func (r *Runtime) StopContainer(c *Container) error {
 	c.opLock.Lock()
 	defer c.opLock.Unlock()
-	if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.path, "kill", c.name); err != nil {
+	if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.path, "kill", c.name, "KILL"); err != nil {
```

Comment:
The old code used to send `SIGKILL`. I'm just making it so that the code actually works. If you want to fix the behaviour, we can discuss that separately.

> If so, it seems more "polite" to send a SIGINT first, to give the application time to clean up,

Did you mean SIGTERM? SIGINT is for `<c-c>` interrupts, and usually isn't handled gracefully by daemons.